controller/src/beerocks/master/db/node.h

```diff
@@ -51,7 +51,8 @@ class node {
 
     std::string ipv4;
     std::string manufacturer;
-    int channel = 0;
+    int channel             = 0;
+    uint8_t operating_class = 0;
```

Comment:
> I'm not sure it is the right place to put it. Because it is radio related data, and where you put it is a place for a common parameters (channel, for example, is common to radio and station nodes)

It probably isn't the right place to put it, but at the moment it is where the channel is, and it makes sense to put the operating class in the same place I think.

Reply:
Moved under radio node. Set\get changed accordingly.